src/transformers/models/beit/configuration_beit.py

```diff
@@ -176,3 +181,20 @@ def __init__(
         self.auxiliary_num_convs = auxiliary_num_convs
         self.auxiliary_concat_input = auxiliary_concat_input
         self.semantic_loss_ignore_index = semantic_loss_ignore_index
+
+
+class BeitOnnxConfig(OnnxConfig):
```

Comment:
I suggest adding the following comment above the class so we can cross-reference any changes on the ViT side :)

```suggestion
# Copied from transformers.models.vit.configuration_vit.ViTOnnxConfig
class BeitOnnxConfig(OnnxConfig):
```

Reply:
Thanks @lewtun! I added the recommended comment.

I haven't done any benchmarking of the original model vs ONNX, but I would be quite curious about that as well. Ultimately we're trying to use ONNX as a stepping stone to converting to TensorRT in the hopes of eking out even more performance, but if you have a benchmarking script that I could borrow I'd be happy to run it against BEiT and share the results. I checked the Transformers benchmarking utility and it doesn't look like it supports image classification models yet.

Reply:
Thanks for the update! We don't have a proper benchmarking script for ONNX models, but we should definitely think about including one in [`optimum`](https://github.com/huggingface/optimum) which is focused on optimisation topics. I've opened an issue here that you can track / comment on: https://github.com/huggingface/optimum/issues/128